Microsoft.Toolkit/Microsoft.Toolkit.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
This scenario is now only being tested by the UWP runner. Should we have .Net Core Tests for this as well? Not more tests, just a .net code2.1 csproj that references the guard tests you already wrote, and maybe just a few for the nullable change. What do you think?

Reply:
Ahahah yup we wrote the same thing at the same time, I was thinking of basically just replicating the same setup as the one in the `HighPerformance` tests, with a shared project and then a UWP project and a .NET Core 2.1/3.1 unit test project as well for the shared APIs. 😄

Should I go ahead and set that up in this same PR?

Reply:
Absolutely! Perfect!